src/parse-options.js

```diff
@@ -1,18 +1,33 @@
 import {resolve} from 'path'
+import styles from 'chalk'
 
 const DEFAULT_OPTIONS = {
   nodeBin: 'node',
   maxAttempts: 3,
   protractorArgs: [],
+  // set color to true to use a default color or set to one of the colors available at 'chalk' - https://github.com/chalk/ansi-styles#colors
+  color: null,
   // the name of one of the included parsers
   // a function to be used as a parser
   // or the path to a node module that exports a parser
   parser: 'standard'
 }
 
+function OptionsException (message) {
```

Comment:
I don't think there's a need for a new type of Exception here. I'm fine with just `new Error`